src/plasoscaffolder/bll/mappings/mapping_helper.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Helper methods for mapping"""
+import os
+
+from jinja2 import Environment
+from jinja2 import FileSystemLoader
+
+
+def _get_template_path() -> os.path:
+  """
+  the path to the template file
+  :param file_name: the name of the file
+  :return: the template file path
+  """
+  path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
instead of using __file__ (which can change see: https://github.com/log2timeline/l2tdevtools/blob/f06804081de5777a1c3b4a09aa7150f279a7ab81/tools/build.py#L19) 

maybe pass templates_path as a function argument, potentially the user could pass this as an argument.